src/bindings/channel_header.ts

```diff
@@ -1,20 +1,20 @@
 import {AppBinding, AppContext} from 'mattermost-redux/types/apps';
 import {AppExpandLevels} from 'mattermost-redux/constants/apps';
 
-import {getStaticURL, Routes, newChannelHeaderBindings} from '../utils';
+import {getStaticURL, Routes, newChannelHeaderBindings, isConfigured, isConnected, isUserSystemAdmin} from '../utils';
 import {CommandLocations, ZendeskIcon} from '../utils/constants';
 
 // getChannelHeaderBindings returns the users command bindings
```

Comment:
s/command/channel header.

Reply:
done.  I renamed CommandLocations to Locations this is agnostic.  This was left over from when I initially created the app with only command locations